include/elliptics/session.hpp

```diff
@@ -317,7 +317,7 @@ class session
 		 * in this session so they won't collide with same key in any
 		 * other namespace.
 		 */
-		void set_namespace(const std::string &ns);
+		void			set_namespace(const std::string &ns);
```

Comment:
Let's make all consistent and will use one space for all methods and tabs only for members?